### DISCUSSION TITLE
after press "clean history", the coapp version is not included in the report

### DISCUSSION THREAD

**Discussion by trup** [aclap-dev/video-downloadhelper#1158] [Help and Q&A]
### Your question

To fix it, you need to click "recheck" in the settings.

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: ru
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "used_history_button": true,
    "download_directory": "C:\\Users\\AnnA\\Videos",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\AnnA\\Videos\\новости"
        }
    },
    "concurrent_downloads_max": 5,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": true
    },
    "show_context_menu": false,
    "yt_warning": false,
    "use_wide_ui": false,
    "use_legacy_ui": false,
    "use_sidebar": false,
    "open_count_store": 34,
    "successfull_dl": 8,
    "never_show_successfull_dl_message": true,
    "record_download_history": true
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I was about to agree with you but then I couldn't replicate the problem.  Right after I read this, my information didn't include the CoApp version.  Another user posted something about a problem with a YouTube video so I cleared my history, then downloaded the YouTube video.  My `Copy information to clipboard` information included my CoApp version.  Then I cleared history & copied the information to my clipboard again.  It still showed my CoApp version.

This might be a one-off issue.  It might be that the first time you clear your history, the button forgets to copy the CoApp version.  But after a `Reload extension` (what I did instead of a `Recheck`), the CoApp version seems to stick.

I find it interesting that what gets copied to the clipboard includes your VDH history.  That seems rather wasteful.  It threatens to make the things folks post here for their information grow rather large with items that might not be all that useful or interesting.  Of course, clearing your history shortens the amount of data copied to the clipboard.
